cartridges/app_storefront_base/cartridge/controllers/Home.js

```diff
@@ -22,26 +23,34 @@ var pageMetaData = require('*/cartridge/scripts/middleware/pageMetaData');
  * @param {renders} - isml
  * @param {serverfunction} - get
  */
-server.get('Show', consentTracking.consent, cache.applyDefaultCache, function (req, res, next) {
-    var Site = require('dw/system/Site');
-    var PageMgr = require('dw/experience/PageMgr');
-    var pageMetaHelper = require('*/cartridge/scripts/helpers/pageMetaHelper');
-
-    pageMetaHelper.setPageMetaTags(req.pageMetaData, Site.current);
-
-    var page = PageMgr.getPage('homepage');
-
-    if (page && page.isVisible()) {
-        res.page('homepage');
-    } else {
-        res.render('home/homePage');
-    }
-    next();
-}, pageMetaData.computedPageMetaData);
-
-server.get('ErrorNotFound', function (req, res, next) {
+server.get(
+    "Show",
+    // userLoggedIn.validateLoggedIn,
```

Comment:
Please avoid committing commented code

Reply:
Yes, I removed them from the other PR, but commented code still remains in that PR